src/python/WMCore/MicroService/MSTransferor/DataStructs/Workflow.py

```diff
@@ -484,3 +484,25 @@ def getOpenRunningTimeout(self):
         :return: an integer with the amount of secs
         """
         return self.data.get("OpenRunningTimeout", 0)
+
+    def getInputData(self):
```

Comment:
You don't need this method implementation here since it is defined in `Workflow` class and since it is identical it can be simply dropped. This method will be take from a parent class.